samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -71,21 +109,68 @@ static void get_data(struct k_work *work)
 			printk("Error getting relative chip temperature data (%d)\n",
 			       err);
 		}
-	} else {
+		break;
+	}
+	case (2): {
 		err = bt_mesh_sensor_cli_get(
 			&sensor_cli, NULL, &bt_mesh_sensor_time_since_presence_detected,
 			NULL);
 		if (err) {
 			printk("Error getting time since presence detected (%d)\n",
 			       err);
 		}
+		break;
 	}
+	}
+	k_work_schedule(&get_data_work, K_MSEC(GET_DATA_INTERVAL));
+}
 
-	req_temp_range = !req_temp_range;
+static const struct sensor_value temp_ranges[][2] = {
+	{ { 0 }, { 100 } },
+	{ { 10 }, { 20 } },
+	{ { 22 }, { 30 } },
```

Comment:
Should it be ...
```suggestion
	{ { 20 }, { 30 } },
```
?